mslicedevel.sh

```diff
@@ -3,5 +3,5 @@
 # Basic wrapper script for running in development mode. It assumes the current
 # working directory is the directory containing this script.
 #
-env QT_API=pyqt python setup.py build_py --inplace
+env QT_API=pyqt python setup.py
```

Comment:
We shouldn't need this entire line now as there's no setup to run.

Reply:
Then do we still need setup.py?

Reply:
It's currently used to run flake8 & nosetests so that it reads the `setup.cfg` so we'll need to keep it. In the future we'll use it for packaging too.